.github/workflows/bump-version.yml

```diff
@@ -6,10 +6,14 @@ on:
       - master
     paths-ignore:
       - HISTORY.rst
+      - Makefile
+      - MANIFEST.in
+      - requirements_upstream.txt
       - setup.cfg
       - setup.py
       - tox.ini
       - xclim/__init__.py
+      - docs/**.rst
```

Comment:
What does this fix?
So, if I understand correctly, this means we bump the version only when the code changes, but not for a doc PR.

Reply:
Precisely. This will trigger if a notebook changes, though. Should I exclude both?

Reply:
Ok! Bah, I don't think I care that much about the difference, but indeed excluding the notebooks would fit the idea.